src/gebt_poc/mesh.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include <Kokkos_Core.hpp>
+
+namespace openturbine {
+class Mesh {
+public:
+    int GetNumberOfElements() { return number_of_elements_; }
+
+    int GetNumberOfNodes() { return number_of_nodes_; }
+
+    void SetNumberOfElements(int number_of_elements) { number_of_elements_ = number_of_elements; }
+    void SetNumberOfNodes(int number_of_nodes) { number_of_nodes_ = number_of_nodes; }
+
+    void InitializeElementNodeConnectivity(int number_of_elements, int nodes_per_element) {
```

Comment:
I've moved these functions into the protected namespace and actually plan to do so with the default constructor (I thought I had already done that).  I want to avoid depending on the details of how I've implemented this mesh except in very intentional locations.  It handles the simple case of beam elements with a fixed number of nodes very well, but will need to be changed quite a bit to do other things.  It's better to control the leakage of internal structure now rather than have to clean things up in the future.